packages/graphql/src/schema/resolvers/count.ts

```diff
@@ -34,7 +34,12 @@ export default function countResolver({ node }: { node: Node }) {
             raw: true,
         });
 
-        return result.records[0]._fields[0].toNumber();
+        const count: number | any = result.records[0]._fields[0];
+
+        if (typeof count === "number") {
+            return count;
+        }
+        return count.toNumber();
```

Comment:
Damn, this is definitely needed here, sorry about that! We actually do this slightly differently elsewhere in the codebase and would like to stick with that for consistency:

```suggestion
        const count = result.records[0]._fields[0];

        // @ts-ignore: count is unknown, and to cast to object would be an antipattern
        if (isInt(count)) {
            return (count as Integer).toNumber();
        }

        return count;
```

Which will also require the following imports at the top of the file:

```ts
import { isInt, Integer } from "neo4j-driver";
```

Reply:
Agreed, changed that part.